src/module/cp/atc.py

```
@@ -1,59 +1,91 @@
+                           aliases={('atcoder', 'contest'), ('atc', 'contests'), ('atcoder', 'contests')}, priority=50,
+                           block=True)
+
+random_pick_query = on_command(('atc', 'pick'),
+                               aliases={('atcoder', 'pick'), ('atc', 'prob'), ('atcoder', 'prob'), ('atc', 'problem'),
+                                        ('atcoder', 'problem')}, priority=50, block=True)
+
+help_query = on_command('atc', aliases={'atcoder'}, priority=500, block=True)
+
+
+@id_query.handle()
+async def send_user_id_card(event: Event, message: Message = CommandArg()):
```

Ruff (0.12.2)

Do not perform function call `CommandArg` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)